api/api/middleware/firebaseAuth.go

```diff
@@ -29,9 +30,8 @@ func InitFirebaseClient() *auth.Client {
 	if firebaseClient != nil {
 		return firebaseClient
 	}
-	// added
 
-	opt := option.WithCredentialsFile("/home/speedy/Desktop/Roommate_Finder/cribconnect/api/api/middleware/roommate-finder-6f8ef-firebase-adminsdk-fbsvc-463d6d62c8.json")
+	opt := option.WithCredentials(&google.Credentials{})
```

Comment:
Consider adding a comment here since it's doing some magic here that's not obviously.
```suggestion
	// google.Credentials will search for credentials in a number of default locations including the FIREBASE_CONFIG environment variable.
	opt := option.WithCredentials(&google.Credentials{})
```